examples/model_assisted_labeling/tiled_imagery_mal.ipynb

```diff
@@ -35,34 +35,34 @@
   },
```

Comment:
<p>Can you sort these? Also if you can import from labelbox.data.annotation_types then just import all of them at once. Most of this can be imported from https://github.com/Labelbox/labelbox-python/blob/develop/labelbox/data/annotation_types/__init__.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>